src/runner/main.cpp

```diff
@@ -290,6 +294,11 @@ int WINAPI WinMain(HINSTANCE hInstance, HINSTANCE hPrevInstance, LPSTR lpCmdLine
     {
         return 0;
     }
+
+    std::filesystem::path logFilePath(PTSettingsHelper::get_root_save_folder_location());
+    logFilePath = logFilePath.append(L"runner-logging.txt");

```

Comment:
This should be called `runner-log.txt` and it would be better to have these strings (see line below) defined as constants.